### ISSUE TITLE
subject or participant?

### ISSUE THREAD

**Issue by jdkent** [HBClab/NiBetaSeries#12]
I should conform the function names and doc strings to use one word or the other. It's typically considered more polite say participants, but bids uses the subject vernacular.

**Comment by PeerHerholz** [comment]
Very important point IMHO. You summarized it perfectly: more polite to say participants but most resources, such as BIDS, use subject. 
[BIDS apps](https://bids-apps.neuroimaging.io) also actually combine it: BIDS structure note "subject" but e.g., the [docker command of mriqc](http://mriqc.readthedocs.io/en/latest/running.html#Options%20for%20filtering%20BIDS%20queries) notes "participant". So maybe it's still possible to have both. Otherwise I would say adapt to BIDS is easier on the long run.

**Comment by jdkent** [comment]
yeah, I think this issue should be propagated to the bids standard.
